docs/Contributing/getting-started/README.md

```diff
@@ -0,0 +1,28 @@
+# Getting Started with Fleet Development
+
+This directory contains documentation to help you get started with Fleet development.
+
+## Setting Up Your Development Environment
+
+- [Building Fleet](building-fleet.md) - Instructions for building Fleet from source
+- [Testing and Local Development](testing-and-local-development.md) - Guide for testing and local development
+- [Run Locally Built Fleetd](run-locally-built-fleetd.md) - Guide for running a locally built fleetd
+
+## Next Steps
+
+Once you have your development environment set up, you can explore the following resources:
+
+- [Guides](../guides/README.md) - How-to guides for common tasks
```

Comment:
Should we link to READMEs here or just to the base directory for each? If we're assuming GitHub is how folks will be reading the readmes, linking to the directory rather than the readme file give you both the directory tree at the top and readme at the bottom, which helps discoverability a little bit.

If we're optimizing for something else (e.g. Unblocked crawling), fair enough.

Reply:
For now I'm optimizing for browsing via markdown but open to changing that later.